elementary/monitor/api/tests/tests.py

```diff
@@ -186,120 +165,118 @@ def get_test_results(
+
         test_runs = defaultdict(list)
-        for test_metadata in test_results_metadata:
-            test_metadata_dict = dict(test_metadata)
-            test_sub_type_unique_id = self.get_test_sub_type_unique_id(
-                **test_metadata_dict
+        for raw_test_result in latest_test_results:
+            test_invocations = tests_invocations.get(
+                raw_test_result.elementary_unique_id
             )
-            test_invocations = tests_invocations.get(test_sub_type_unique_id)
             test_run = TestRunSchema(
-                metadata=self.get_test_info_from_test_metadata(test_metadata),
+                metadata=self.get_test_metadata_from_raw_test_result(raw_test_result),
                 test_runs=test_invocations,
             )
-            test_runs[test_metadata.model_unique_id].append(test_run)
+            test_runs[raw_test_result.model_unique_id].append(test_run)
 
         self.set_run_cache(key=TEST_RUNS, value=test_runs)
         return test_runs
 
-    def get_invocations(
-        self, invocations_per_test: int = 720, days_back: Optional[int] = 7
+    def _get_invocations(
+        self, raw_test_results: List[RawTestResultSchema]
     ) -> Dict[TestUniqueIdType, InvocationsSchema]:
-        run_operation_response = self.dbt_runner.run_operation(
-            macro_name="get_tests_invocations",
-            macro_args=dict(
-                invocations_per_test=invocations_per_test, days_back=days_back
-            ),
-        )
-        test_invocation_dicts = (
-            json.loads(run_operation_response[0]) if run_operation_response else []
-        )
         grouped_invocations = defaultdict(list)
-        for test_invocation in test_invocation_dicts:
+        for raw_test_result in raw_test_results:
             try:
-                sub_test_unique_id = self.get_test_sub_type_unique_id(**test_invocation)
-                grouped_invocations[sub_test_unique_id].append(
+                elementary_unique_id = raw_test_result.elementary_unique_id
+                grouped_invocations[elementary_unique_id].append(
                     InvocationSchema(
-                        id=test_invocation["test_execution_id"],
-                        time_utc=test_invocation["detected_at"],
-                        status=test_invocation["status"],
+                        id=raw_test_result.invocation_id
+                        or raw_test_result.test_execution_id,
+                        time_utc=raw_test_result.detected_at,
+                        status=raw_test_result.status,
                         affected_rows=self._parse_affected_row(
-                            results_description=test_invocation[
-                                "test_results_description"
-                            ]
+                            results_description=raw_test_result.test_results_description
                         ),
                     )
                 )
             except Exception:
```

Comment:
Why do we have an `except Exception` here?

Reply:
This is here for a long time.
In case we couldn't parse an invocation, we didn't want to fail the whole report.
Now we are braver and changed our approach for breaking the report, but I am not sure if we want to remove that completely an fail on a single invocation 🤔 

My hunch is to remove it, but want to make sure first